ballista/rust/executor/src/standalone.rs

```diff
@@ -36,40 +37,43 @@ pub async fn new_standalone_executor(
     scheduler: SchedulerGrpcClient<Channel>,
     concurrent_tasks: usize,
 ) -> Result<()> {
-    let work_dir = TempDir::new()?
-        .into_path()
-        .into_os_string()
-        .into_string()
-        .unwrap();
-    let executor = Arc::new(Executor::new(&work_dir));
-
-    let service = BallistaFlightService::new(executor.clone());
-
-    let server = FlightServiceServer::new(service);
     // Let the OS assign a random, free port
     let listener = TcpListener::bind("localhost:0").await?;
     let addr = listener.local_addr()?;
     info!(
         "Ballista v{} Rust Executor listening on {:?}",
         BALLISTA_VERSION, addr
     );
-    tokio::spawn(
-        Server::builder().add_service(server).serve_with_incoming(
-            tokio_stream::wrappers::TcpListenerStream::new(listener),
-        ),
-    );
+
     let executor_meta = ExecutorRegistration {
         id: Uuid::new_v4().to_string(), // assign this executor a unique ID
         optional_host: Some(OptionalHost::Host("localhost".to_string())),
         port: addr.port() as u32,
         // TODO Make it configurable
         grpc_port: 50020,
```

Comment:
Please do not hard code the grpc_port, please make it configurable otherwise we will be unable to startup multiple executor instances in one nodes.

Reply:
It's mainly for unit test. For production execution, it's already configurable.

Reply:
Note that this PR doesn't introduce the hard coded port -- perhaps it is worth a ticket to track making the value configurable

Reply:
I patched up the compile errors in https://github.com/apache/arrow-datafusion/pull/1839 -- looks like a test is also failing https://github.com/apache/arrow-datafusion/issues/1840

Reply:
Hi @alamb and @thinkharderdev, should we fix the standalone ut issue here or by another PR?

Reply:
@yahoNanJing @alamb already pushed a PR to address the compilation issue https://github.com/apache/arrow-datafusion/pull/1839 so you're good.